blocks/extensions/predicates.py

```diff
@@ -12,3 +12,7 @@ def __init__(self, record_name):
 
     def __call__(self, log):
         return bool(log.current_row.get(self.record_name, False))
+
+    def __eq__(self, other):
```

Comment:
Specifically, https://github.com/mila-udem/blocks/pull/1109/files#diff-376409566e985ed805d621c4bf885535R183

I can imagine it being useful in other contexts too...